### ISSUE TITLE
Navmesh files for Scannet

### ISSUE THREAD

**Issue by shaantanu314** [facebookresearch/habitat-sim#1607]
## Habitat-Sim version
v0.2.1

I need to get navmesh files for scannet. where can I get those. I'm working on scannet dataset and following warning occurs:

> Could not find navmesh dataset/glb/scene0742_00_vh_clean_2.navmesh, no collision checking will be done

The following is the complete error log
[pastebin link](https://pastebin.com/VMap65NB)

**Comment by aclegg3** [comment]
If the dataset does not come with NavMesh files, you can create and save your own from within the Habitat simulator. Check out our in-depth tutorial on the navmesh ([`Habitat-Sim Basics for Navigation`](https://colab.research.google.com/github/facebookresearch/habitat-sim/blob/main/examples/tutorials/colabs/ECCV_2020_Navigation.ipynb) from [ECCV 2020 series](https://aihabitat.org/tutorial/2020/)) for detailed example code. 

TL;DR: 
1. load the scene
2. [recompute_navmesh](https://aihabitat.org/docs/habitat-sim/habitat_sim.simulator.Simulator.html#recompute_navmesh)
3. [save_nav_mesh](https://aihabitat.org/docs/habitat-sim/habitat_sim.nav.PathFinder.html#save_nav_mesh) for later re-load
